cdap-explore-jdbc/src/main/java/co/cask/cdap/explore/jdbc/ExploreStatement.java

```diff
@@ -43,7 +43,7 @@
 public class ExploreStatement implements Statement {
   private static final Logger LOG = LoggerFactory.getLogger(ExploreStatement.class);
 
-  private int fetchSize = 50;
+  private int fetchSize = 1000;
```

Comment:
Even if we take each row to be on an average 100 KB, fetching 1000 rows at a time would need about 100 MB of memory for each fetch. So I think this is okay.